packages/flutter/lib/src/material/pickers/calendar_date_picker.dart

```diff
@@ -599,14 +741,27 @@ class _MonthPickerState extends State<_MonthPicker> {
               ],
             ),
           ),
-          _DayHeaders(),
           Expanded(
-            child: PageView.builder(
-              controller: _pageController,
-              itemBuilder: _buildItems,
-              itemCount: utils.monthDelta(widget.firstDate, widget.lastDate) + 1,
-              scrollDirection: Axis.horizontal,
-              onPageChanged: _handleMonthPageChanged,
+            child: Actions(
+              actions: _actionMap,
+              child: Focus(
+                focusNode: _dayGridFocus,
+                onFocusChange: _handleGridFocusChange,
+                child: _FocusedDate(
+                  date: _dayGridFocus.hasFocus ? _focusedDay : null,
+                  child: Container(
+                    color: _dayGridFocus.hasFocus ? Theme.of(context).focusColor : null,
```

Comment:
Should the whole grid be colored for focus vs just applying the color to the focused day?

Reply:
Not sure, but it was how the original Android date picker handled this. As the navigation in the grid is a little different (you only use the tab to go in and out of the grid), it seems like a good way to indicate that.

Reply:
Gotcha, sounds good